src/chess/board.h

```diff
@@ -43,6 +43,7 @@ class ChessBoard {
   ChessBoard(const std::string& fen) { SetFromFen(fen); }
 
   static const std::string kStartingFen;
+  static const ChessBoard kStartBoardPos;
```

Comment:
Why not startposBoard?. Maybe for consistency name them `kStartposFen` and `kStartposBoard`?